Dockerfile

```diff
@@ -24,22 +25,40 @@ RUN mix deps.get --only $MIX_ENV
 RUN mix deps.compile
 
 # build assets
+COPY priv priv
+
+# Note: if your project uses a tool like https://purgecss.com/,
+# which customizes asset compilation based on what it finds in
+# your Elixir templates, you will need to move the asset compilation step
+# down so that `lib` is available.
 COPY assets assets
-RUN rm assets/webpack.config.js
```

Comment:
we need to keep these lines because the webpack docker config needs to replace the regular webpack config due to env vars

Reply:
brought them back 👍 check last commit